packages/eslint-config-hypothesis/jsx.js

```diff
@@ -0,0 +1,7 @@
+import jsxA11y from 'eslint-plugin-jsx-a11y';
+import legacy from './index.js';
+
+export default [
+  ...legacy,
```

Comment:
Why does the JSX config extend legacy instead of base?

Reply:
Basically, we need to end up having this:

* Base:
    * Eslint recommended rules
    * Mocha recommended rules
* Legacy:
    * Eslint recommended rules
    * Mocha recommended rules
    * React recommended rules
    * React hooks recommended rules
* JSX:
    * Eslint recommended rules
    * Mocha recommended rules
    * React recommended rules
    * React hooks recommended rules
    * JSX a11y recommended rules

So I did this "chain" of inheritance to avoid duplicating stuff.

We could make the legacy entry point extend from jsx, but since jsx imports `eslint-config-jsx-a11y`, which is an optional peer dependency, it could introduce a breaking change.